src/test/scala/com/fulcrumgenomics/vcf/DownsampleVcfTest.scala

```diff
@@ -222,15 +221,17 @@ class DownsampleVcfTest extends UnitSpec {
 
   it should "return the same results for biallelic and generalized algorithm" in {
     val e = 0.01
+    // TODO: figure out what the second item in the collection was meant for.
     val cases: IndexedSeq[(IndexedSeq[Int], IndexedSeq[Double])] = IndexedSeq(
       (IndexedSeq(1, 0), IndexedSeq(1 - e, 0.5, e)),
       (IndexedSeq(0, 1), IndexedSeq(e, 0.5, 1 - e)),
       (IndexedSeq(1, 1), IndexedSeq((1 - e) * e, 0.25, (1 - e) * e)),
       (IndexedSeq(2, 0), IndexedSeq(math.pow((1 - e), 2), 0.25, math.pow(e, 2))),
     )
-    cases.foreach { case (input, output) =>
+    cases.foreach { case (input, _) =>
       val biallelic = Likelihoods(2, DownsampleVcf.Likelihoods.biallelic(input(0), input(1), e))
       val generalized = Likelihoods(2, DownsampleVcf.Likelihoods.generalized(input, e))
+      biallelic.pls should contain theSameElementsInOrderAs biallelic.pls
       biallelic.pls should contain theSameElementsInOrderAs generalized.pls
```

Comment:
I think there's two methods for calculating per-allele likelihoods.  The first using a biallelic model, whereas the second uses a multiallelic model (for triallelic and so on).  They should be the same when two alleles are given.  Furthermore, the second item in the list is the _expected_ computed likelihoods, so we'll want to use that.

```suggestion
    cases.foreach { case (alleleDepths, likelihoods) =>
      val biallelic = Likelihoods(2, DownsampleVcf.Likelihoods.biallelic(alleleDepths(0), alleleDepths(1), e))
      val generalized = Likelihoods(2, DownsampleVcf.Likelihoods.generalized(alleleDepths, e))
      // Check that the biallelic likelihoods match
      biallelic.pls should contain theSameElementsInOrderAs likelihoods
      // Check that both methods of computing the likelihoods match
      biallelic.pls should contain theSameElementsInOrderAs generalized.pls
```

Reply:
Unfortunately that's not quite right because the `.pls` are a list of integers but the second element in the test data is a list of doubles. I also tried comparing the doubles to the `Liklihood.genotypeLikelihoods` list which matches the type, but none of the numbers were even close to being correct.

I will have to open an issue to track this since it's out of scope for this PR.

Reply:
https://github.com/fulcrumgenomics/fgbio/issues/1051